docs/operations/mediated_devices_configuration.md

```diff
@@ -1,13 +1,13 @@
 # Mediated devices and virtual GPUs
+## Configuring mediated devices and virtual GPUs
 
 KubeVirt aims to facilitate the configuration of mediated devices on large clusters.
-The new `mediatedDevicesConfiguration` API in KubeVirt CR allows administrators to
-create or remove mediated devices in a declarative way, by providing a list of the desired mediated devices types  that they expect to be configured in the cluster.
-
-It would also be possible to provide a more specific configuration that targets a specific node or a group of nodes directly.
-This can be achieved by using the `nodeMediatedDeviceTypes`, however, it must be used in a combination with `mediatedDevicesTypes`, to override the global
-configuration set in `mediatedDevicesTypes` section. Therefore, using `nodeMediatedDeviceTypes` without the `mediatedDevicesTypes` section is not supported.
+Administrators can use the `mediatedDevicesConfiguration` API in the KubeVirt CR to
+create or remove mediated devices in a declarative way, by providing a list of the desired mediated device types that they expect to be configured in the cluster.
 
+You can also include the `nodeMediatedDeviceTypes` option to provide a more specific configuration that targets a specific node or a group of nodes directly with a node selector.
+The `nodeMediatedDeviceTypes` option must be used in combination with `mediatedDevicesTypes`
+in order to override the global configuration set in the `mediatedDevicesTypes` section.
```

Comment:
I was tempted to move this (and change the related example KubeVirt CR snippet) to the 'Overriding config..' section so that this was more singularly about applying the global config, and then subsequently we show how that can be overridden for specific nodes; I think that will make it easier for new user to follow.

Reply:
I wonder what stopped you? :) 

FWIW, I agree. It would read much better.